zero-thrs-control/src/thrs/control/controllers.py

```diff
@@ -49,92 +52,132 @@ def setpoint(self, value: SetpointUnit):
         self._pid.setpoint = value
 
     def __call__(self, measurement: SetpointUnit, time: datetime) -> ValueUnit:
+        if self._reset_time: #TODO: deal with dt is simulation versus reality. In simulation, passing dt = tick_duration works. In real time, having actual time would be nice..
+            self._reset_time = False
```

Comment:
This is just no-opping?

I think it's fine to forcefully pass a time to simple-pid. In real time `time` would be the start of the control cycle, which is indicative of the "now" as that is the best indication of the recentness of the sensor values. The main issue is that `enable` should also force a time (we could also have the cycler pass a time function instead of just a time, then we could just pass that along to the pid and be done with it)